beta/package.json

```diff
@@ -17,7 +17,8 @@
     "ci-check": "npm-run-all prettier:diff --parallel lint tsc",
     "tsc": "tsc --noEmit",
     "start": "next start",
-    "postinstall": "is-ci || (cd .. && husky install beta/.husky)"
+    "postinstall": "is-ci || (cd .. && husky install beta/.husky)",
+    "check-all": "npm-run-all prettier lint:fix tsc"
```

Comment:
should we use the `--parallel` flag here too?

Reply:
Nope. We shall run it sequentially since we are auto fixing using prettier and linter. We shall first completely prettify and then run lint.